ozaria/site/components/teacher-dashboard/BaseCurriculumGuide/index.vue

```diff
@@ -99,6 +105,12 @@ export default {
 <template>
   <div>
     <div>
+      <button
+        @click="openCapstoneProject"
```

Comment:
can we put it as false or under variable like capstoneModal to not show it in the guide for testing after merge?